database/ydb/README.md

```diff
@@ -0,0 +1,30 @@
+# [YDB](https://ydb.tech/docs/)
+
+`ydb://[user:password@]host:port/database?QUERY_PARAMS`
```

Comment:
```suggestion
`grpc[s]://[user:password@]host:port/database?QUERY_PARAMS`
```

Reply:
Also need to add information about certificates. By default `grpcs` used root certificates. But user can define different certificates through env `YDB_SSL_ROOT_CERTIFICATES_FILE`. Reed more about this https://github.com/ydb-platform/ydb-go-sdk?tab=readme-ov-file#environment-variables-

Reply:
done